rust/template/differential_datalog/program.rs

```diff
@@ -1618,9 +1627,10 @@ impl Program {
 
     /* Profiler thread function */
     fn prof_thread_func(chan: mpsc::Receiver<ProfMsg>, profile: Arc<Mutex<Profile>>) {
+        let mut stats = Statistics::new("stats.csv").expect("TODO");
```

Comment:
How should I handle failure to create this file? Currently I panic on failure. Should I instead continue executing without writing statistics and print a warning?

Should the file be named this or have a different name?